Source/AppScaffolding/LibationScaffolding.cs

```diff
@@ -35,7 +37,7 @@ public static class LibationScaffolding
 		public static ReleaseIdentifier ReleaseIdentifier { get; private set; }
 		public static VarietyType Variety
 			=> ReleaseIdentifier == ReleaseIdentifier.WindowsClassic ? VarietyType.Classic
-			: ReleaseIdentifier.In(ReleaseIdentifier.WindowsAvalonia, ReleaseIdentifier.LinuxAvalonia, ReleaseIdentifier.MacOSAvalonia) ? VarietyType.Chardonnay
+			: Enum.IsDefined(ReleaseIdentifier) ? VarietyType.Chardonnay
```

Comment:
In theory, this will also catch `None`. I don't know if it's possible in practice though.

Reply:
OK, I reworked this a little. I added an OS enum to `Configuration`, and `Configuration.OS`.

The `ReleaseIdentifier` is now a the sum of the `OS`, `Variety` (renamed from `VarietyType`), and `Architecture` enum values.

`SetReleaseIdentifier` only takes `Variety` as an argument and figures out the release ID from that and the OS/arch.